src/pymmcore_widgets/_mda/_util/_labeled_slider.py

```diff
@@ -55,44 +56,46 @@ def __init__(
         self.layout().addWidget(self.play_btn)
         self.layout().addWidget(self.slider)
         self.layout().addWidget(self.current_value)
-        self.setSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
+        self.setSizePolicy(QtWidgets.QSizePolicy.Policy.Expanding, FIXED)
 
         self.slider.valueChanged.connect(self.on_drag_timer)
         self.slider.sliderPressed.connect(self.sliderPressed)
         self.slider.sliderMoved.connect(self.sliderMoved)
         self.slider.sliderReleased.connect(self.sliderReleased)
         self.playing = False
 
-        self.play_timer = QtCore.QTimer(interval=10)
+        self.play_timer = QtCore.QTimer()
```

Comment:
at least on Qt6 [there is no overload](https://doc.qt.io/qt-6/qtimer.html#QTimer) that accepts the interval as the first arg